pages/store/claim.vue

```diff
@@ -69,6 +69,59 @@ const cartId = computed(() => getRouteQuery('cart_id'))
 const claimingToken = computed(() => getRouteQuery('claiming_token'))
 const paymentId = computed(() => getRouteQuery('payment_id'))
 
+const baseLoadingLabels = computed(() => [
+  $t('claim_page_loading_step_1'),
+  $t('claim_page_loading_step_2'),
+])
+
+const finalLoadingLabelVariants = computed(() => [
+  $t('claim_page_loading_step_3'),
+  $t('claim_page_loading_step_3_alt_1'),
+  $t('claim_page_loading_step_3_alt_2'),
+  $t('claim_page_loading_step_3_alt_3'),
+  $t('claim_page_loading_step_3_alt_4'),
+  $t('claim_page_loading_step_3_alt_5'),
+  $t('claim_page_loading_step_3_alt_6'),
+  $t('claim_page_loading_step_3_alt_7'),
+  $t('claim_page_loading_step_3_alt_8'),
+  $t('claim_page_loading_step_3_alt_9'),
+])
+
+const loadingLabelIndex = ref(0)
+const randomVariantIndex = ref(0)
+
+const currentLoadingLabel = computed(() => {
+  if (loadingLabelIndex.value < baseLoadingLabels.value.length - 1) {
+    return baseLoadingLabels.value[loadingLabelIndex.value] || $t('claim_page_loading_label')
+  }
+  else {
+    return finalLoadingLabelVariants.value[randomVariantIndex.value] || $t('claim_page_loading_label')
+  }
+})
+
+const {
+  pause: pauseLoadingLabelAnimation,
+  resume: resumeLoadingLabelAnimation,
+  isActive: isLoadingLabelAnimationActive,
+} = useIntervalFn(() => {
+  if (loadingLabelIndex.value < baseLoadingLabels.value.length - 1) {
+    loadingLabelIndex.value++
+  }
+  else {
+    randomVariantIndex.value = Math.floor(Math.random() * finalLoadingLabelVariants.value.length)
+  }
+}, 5000, { immediate: false })
```

Comment:
as we have so much variants, i think we can shorten the interval 😂
```suggestion
}, 3000, { immediate: false })
```